src/styles/blocks/header.scss

```diff
@@ -0,0 +1,7 @@
+.header {
+  padding: 0 50px;
+  box-sizing: border-box;
```

Comment:
just set `box-sizing` for `*` selector once for the whole project and no need to duplicate it